bin/libast_prereq.sh

```diff
@@ -12,6 +12,8 @@ base_dir=`dirname "$bin_dir"`
 
 PATH=$bin_dir:$PATH
 
+: ${CC:=cc}
```

Comment:
This should be unnecessary but good catch that this script didn't honor the `CC` env var.

Reply:
We are in the process of deprecating ```iffe``` tests that would require this fix, but thanks for pointing it out.